lib/target/generic_cpp/generic_cxx_codegen.cc

```diff
@@ -566,6 +566,21 @@ void GenericCXXCodeGen::RunOnFunction(Function& function) {
         os_ << "odla_SetComputationItem(Comp, ODLA_OPT_BATCH_SIZE, "
                "(odla_item_value) &opt_batch_size);\n";
       }
+      if (opts_.bf16_mode == CodeGen::BF16Mode::Accuracy) {
+        os_ << "odla_bf16_mode mode = BF16_ACCURACY_MODE;\n";
+        os_ << "odla_SetComputationItem(Comp, ODLA_BF16_MODE, "
```

Comment:
line 571, 575, 579 are the same.
How about something like 

os_ << "odla_bf16_mode mode = " << get_bf16_mode(opts_bf16_mode) << ";\n";
os_ << "odla_SetComputationItem(Comp, ODLA_BF16_MODE, (odla_item_value) &mode);\n"